### PR TITLE
Support jinja extensions

### LINKED ISSUE #65: Support jinja extensions
As we start to move some more complex rendering logic into our TOC templates I've found some cases where I want to use jinja extensions.  As @frankduncan pointed out when talking to him about this, we don't want to force ALL users of torque to use a predetermined set of extensions, so this means not just adding extensions but adding support for configurable extensions.

### PR DESCRIPTION
**Pull request by slifty**
This PR adds support for configurable jinja extensions.

Resolves #65

### CODE REVIEW

File: torquedata/config.py.tmpl [hunk 1]
```diff
@@ -7,9 +7,12 @@ POSTGRES_DB_PASSWORD="__DATABASE_PASSWORD__"
 # https://docs.djangoproject.com/en/3.1/ref/settings/#std:setting-SECRET_KEY
 SECRET_KEY="__RANDOM_SECRET_KEY__"
 
+# A list of jinja2 extensions to enable when rendering templates
```
Comment (frankduncan) [inline torquedata/config.py.tmpl:10]: I think the only thing I'd need here is an example, or a link to somewhere with what exactly belongs in this list.  Though maybe this is obvious if you have actually done this ever, and my ignorance is the problem!

File: torquedata/core/jinja_factory.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from jinja2 import Environment
```
Comment (frankduncan) [inline torquedata/core/jinja_factory.py:1]: I probably would have just put all of this in utils.py, unless you can see this file growing substantially bigger.

Though maybe we should really split utils into many smaller files so the imports are cleaner in the target directories.

**Review by frankduncan — APPROVED** [review]
Two change suggestions that are completely optional, and otherwise this looks great to me!  Thanks for the improvement!

File: torquedata/config.py.tmpl (hunk 1 above)
Comment (slifty) [reply torquedata/config.py.tmpl:10]: Great call -- adding now

File: torquedata/core/jinja_factory.py (hunk 2 above)
Comment (slifty) [reply torquedata/core/jinja_factory.py:1]: Also a good call!  I'll do that as well (splitting utils might make sense but in the mean time I will stick with the pattern of the project!)

**Pull request merged**